ansible_base/rest_filters/rest_framework/field_lookup_backend.py

```diff
@@ -143,6 +141,17 @@ def value_to_python(self, model, lookup, value):
             value = self.value_to_python_for_field(field, value)
         return value, new_lookup, needs_distinct
 
+    def reserved_names(self, view):
+        """The names in query_params to ignore given the current settings and current view"""
+        from django.conf import settings
+
+        reserved_set = set(settings.ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES)
+
+        if hasattr(view, 'rest_filters_reserved_names'):
+            reserved_set |= set(view.rest_filters_reserved_names)
+
+        return reserved_set
```

Comment:
I always find the operators hard to read for some reason and prefer using the `.` functions on them i.e. 
```
settings_reserved_names = set(getattr(settings, 'ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES', []))
view_override_reserved_naames = set(getattr(view, 'rest_filters_reserved_names', []))
return settings_reserved_names.union(view_override_reserved_naames)
```

Reply:
Personally I struggle with the _named methods_ because it is hard to know if they return the value or operate in-place.

https://docs.python.org/2/library/sets.html#set-objects

```python
>>> s = set(range(5))
>>> s.union(set(range(3, 7)))
{0, 1, 2, 3, 4, 5, 6}
>>> s.update(set(range(3, 7)))
```

The first prints and leaves `s` unchanged, but the second doesn't, it just updates `s` in memory.

It doesn't even feel like there's a formal "name" for the in-place group of methods vs the returning methods. But from the docs, they seem to pretty consistently end in `update`... provided it is a set-to-set operation. IMO that's a lot of qualifiers to expect someone to generally remember. But this is me griping about python in general.

Here, since `reserved_set` is artificially created, I think `reserved_set.update(view.rest_filters_reserved_names)` (works for iterables, as per docs) is would be the most consistent thing. Otherwise, if you want the returning methods, we should be using `frozenset`s for everything! To make these objects disposable.